frontend/Dockerfile

```diff
@@ -0,0 +1,62 @@
+######
+# Use an official Node.js runtime as a base image
+######
+FROM node:18-alpine AS base
```

Comment:
Is there any reason to keep with 18 version?

Reply:
No, @endersonmaia ! You can change it for the version you think will be better. 😉

Reply:
@claudioantonio I believe you have the permission to can create a branch on https://github.com/crypto-bug-hunters/bugless. If not, please notify me. This way, @endersonmaia can commit any changes he sees necessary.